scalableminds/webknossos
# Check team and project name validity

### URL of deployed dev instance (used for testing):
- [checkteamnamevalidity](https://checkteamnamevalidity.webknossos.xyz)

### Steps to test:
- got to Administration - Teams
- try creating a new team
- when a invalid name is entered, the request should not be send and the input field should visualize the misspelling

### Issues:
- fixes [#2591](https://github.com/scalableminds/webknossos/issues/2591)

------
- [x] check team name in frontend
- [x] eventually change the check of project names in frontend
